tests/functional/builtins/codegen/test_abi_decode.py

```diff
@@ -486,11 +490,12 @@ def test_abi_decode_arithmetic_overflow(env, tx_failed, get_contract):
     # test based on GHSA-9p8r-4xp4-gw5w:
     # https://github.com/vyperlang/vyper/security/advisories/GHSA-9p8r-4xp4-gw5w#advisory-comment-91841
     # buf + head causes arithmetic overflow
-    code = """
+    buffer_sz = 32 * 3
```

Comment:
maybe either bufsz or buffer_size?

Reply:
resolved in: https://github.com/vyperlang/vyper/pull/4096/commits/c64eea647158911a9b9c6c5429a8690dfd55df1c